modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -1566,6 +1741,23 @@ private String _getEndpoint(
 			objectEntryId, "?nestedFields=", objectRelationship.getName());
 	}
 
+	private String _getEndpointNestedObjectEntry(
```

Comment:
Since this helper is only used once, let’s inline its logic. If we do need it separately, please rename it to _getEndpoint to match our other methods.